index.d.ts

```diff
@@ -0,0 +1,129 @@
+/** 
+ * Typings for twitter-lite 
+ * 
+ * @version 0.9.4
+ * @author Floris de Bijl <@fdebijl>
+*/
+
+/// <reference types="node" />
+
+import { EventEmitter } from 'events';
+import { Header } from 'oauth-1.0a';
+
+declare namespace TwitterLite {
+  type TwitterOptions = {
+    /** "api" is the default (change for other subdomains) */
+    subdomain: string;
+    /** version "1.1" is the default (change for other subdomains) */
+    version: string;
+    /** consumer key from Twitter. */
+    consumer_key: string;
+    /** consumer secret from Twitter */
+    consumer_secret: string;
+    /** access token key from your User (oauth_token) */
+    access_token_key: string;
+    /** access token secret from your User (oauth_token_secret) */
+    access_token_secret: string;
+  };
+
+  type AccessTokenOptions = {
+    key: string;
+    secret: string;
+    /** If using the OAuth web-flow, set this parameter to the value of the oauth_verifier returned in the callback URL. If you are using out-of-band OAuth, set this value to the pin-code. */
+    verifier: string | number;
+  }
+
+  type BearerResponse = {
+    token_type: 'bearer';
+    access_token: string;
+  }
+
+  type RequestTokenResponse = {
+    oauth_token: string;
+    oauth_token_secret: string;
+  }
+
+  type AccessTokenResponse = {
+    oauth_token: string;
+    oauth_token_secret: string;
+    user_id: number;
+    screen_name: string;
+  }
+
+  class Twitter {
+    private authType: 'App' | 'User';
+    private url: string;
+    private oauth: string;
+    private config: TwitterOptions;
+    private client: any;
```

Comment:
Could we type this? I'm looking at the `oauth-1.0a` library now and they don't have types, but I think we could at least do something like this, right? Maybe when they add types later on we can come back and change this.

If you're curious: https://github.com/ddo/oauth-1.0a/blob/8c24a413ab36c7cd049d34a3d2d16996f24da0ad/oauth-1.0a.js#L9.

Note that I haven't done much work with TypeScript so please correct me if I'm wrong 😄 

```typescript
type OAuthClient = object
...
private client: OAuthClient
```